aptos-move/framework/aptos-stdlib/sources/copyable_any.spec.move

```diff
@@ -6,18 +6,27 @@ spec aptos_std::copyable_any {
 
     spec pack<T: drop + store + copy>(x: T): Any {
         use std::bcs;
+        use aptos_std::from_bcs;
         aborts_if false;
+        pragma opaque;
         ensures result == Any {
             type_name: type_info::type_name<T>(),
             data: bcs::serialize<T>(x)
         };
+        ensures [abstract] from_bcs::deserializable<T>(result.data);
```

Comment:
This is OK because of we had this axiom:[ axiom<T> forall v: T: deserializable<T>(bcs::serialize(v)); ](https://github.com/aptos-labs/aptos-core/blob/c3c9effa76cabb075a58dd34cbd43bdb1058a1e7/aptos-move/framework/aptos-stdlib/sources/from_bcs.spec.move#L30)